src/main/resources/swagger/api-docs.yaml

```diff
@@ -1704,6 +1704,31 @@ paths:
               requiresRefresh:
                 type: boolean
 
+  /api/sharelog/sharees:
+    get:
+      tags:
+      - ShareLog
+      operationId: getSharees
+      summary: |
+        Get a list of users with whom current user has shared a Workspace, Group, or Method.
+      description: |
+        Get a list of users with whom current user has shared a Workspace, Group, or Method.
+      parameters:
+        - name: shareType
+          required: false
+          description: |
+           The type of share - Workspace, Group, or Method. If left empty, all types are returned.
+          type: string
+      responses:
+        200:
+          description: Success
+          schema:
+            $ref: '#/definitions/StringArray'
+        404:
+          description: Not Found
```

Comment:
what conditions cause the 404? i.e. what do we return if the current user has no entries in the log? I suggest this should return a 200 OK with a payload of `[]` instead of a 404.